.eslintrc.json

```diff
@@ -11,5 +11,11 @@
     "jsx-a11y/no-autofocus": 0,
     "import/no-named-as-default": 0,
     "max-len": 0
+  },
+  "globals": {
```

Comment:
Add an extending `eslintrc.json` inside the `test/` folder, these are not valid globals in `src/`